test/scripts/kind_cluster.yaml

```diff
@@ -17,8 +17,8 @@ nodes:
     protocol: TCP
   - containerPort: 7444 # flightctl agent grpc endpoint API
     hostPort: 7444
-  - containerPort: 8080 # keycloak
-    hostPort: 8080
+  - containerPort: 8081 # keycloak
+    hostPort: 8081
```

Comment:
please update it here too: 
https://github.com/flightctl/flightctl/blob/main/deploy/helm/flightctl/values.dev.yaml#L14